packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -423,6 +426,19 @@ class Tabs extends Foundation<TabsHandledProps, TabsUnhandledProps, TabsState> {
               );
     }
 
+    /**
+     * Need to filter out none truthy results for Preact.
+     * Can remove if below gets merged in.
+     * https://github.com/preactjs/preact-compat/pull/461
+     */
+    private filterChildren(nodes: React.ReactNode): React.ReactNode {
+        if (Array.isArray(nodes)) {
+            return nodes.filter(Boolean);
+        } else {
+            return undefined;
```

Comment:
Sorry we had a misunderstanding - I think you just want to return `nodes` here because there is nothing to filter.